src/components/pages/pages.spec.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable react/prop-types */
 import React, { useState } from "react";
-import { shallow, mount } from "enzyme";
+import { shallow, mount, ReactWrapper, ShallowWrapper } from "enzyme";
 import { act } from "react-dom/test-utils";
 import BasePages, { Page } from "./pages.component";
```

Comment:
```suggestion
import Pages, { Page } from "./pages.component";
```
What do you think about changing the name to `Pages` like you've done in our stories?